doc/TockBinaryFormat.md

```diff
@@ -201,6 +203,12 @@ will be padded with up to 3 bytes. Each element begins with a 16-bit type and
 TBF may contain arbitrary element types. A standard set of element types are
 standardized.
 
+The standard set of element types shall have its highest bit (bit 15)
+of `Type` unset.
```

Comment:
"standard" is not defined here. The closest sentence to a definition ("A standard set of element types are standardized") needs a rewrite. This makes this section somewhat vague.

How about:

TBF may contain arbitrary element types. To avoid type ID collisions between elements defined by the Tock project and elements defined out-of-tree, the ID space is partitioned into two segments. Type IDs defined by the Tock project will have their high bit (bit 15) unset, and type IDs defined out-of-tree should have their high bit set.

Reply:
Thanks for the suggestion. I've applied it. Please review.